azurerm/data_source_virtual_network.go

```diff
@@ -29,6 +29,14 @@ func dataSourceArmVirtualNetwork() *schema.Resource {
 				},
 			},
 
+			"address_space": {
```

Comment:
we should also deprecate the old `address_spaces` field, and add that to the deprecations page

Reply:
👍